sessions/slides/introduction-to-ensembles.qmd

```diff
@@ -99,4 +106,8 @@ Many uncertainties, many approaches: many models
 
 [Return to the session](../forecast-ensembles)
 
+## References
+
+::: {#refs}
+:::
```

Comment:
I think if we add the markdown ref it might work as a link in which case we don't need this - that said I guess there's no harm in having it

Reply:
I will just leave it in as I think it links to here doesn't it?

Reply:
Oh is that how it works? Makes sense.